File: KarateKlub/KarateKlub.Api/Models/RezultatiTakmicenjaClanPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{

    public class RezultatiTakmicenjaClanPregledVM
    {
        public class Row
        {
            public string nazivTakmicenja { get; set; }
            public string mjestoOdrzavanja { get; set; }
            public DateTime datumOdrzavaja { get; set; }
            public string osvojenoMjesto { get; set; }
            public string kategorija { get; set; }
            public string uzrast { get; set; }
            public string disciplina { get; set; }
        }
        public List<Row> rows { get; set; }
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_clan/model/RezultatiTakmicenjaClanPregledVM.java
package com.example.x.karateklub.area_clan.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class RezultatiTakmicenjaClanPregledVM implements Serializable {
    public static class Row implements Serializable
    {
        public String nazivTakmicenja;
        public String mjestoOdrzavanja;
        public Date datumOdrzavaja;
        public String osvojenoMjesto;
        public String kategorija;
        public String uzrast;
        public String disciplina;
    }

        public List<Row> rows;

}
